src/content/docs/ko/core-concepts/sharing-state.mdx

```diff
@@ -0,0 +1,824 @@
+- Nano Stores를 클라이언트 측 컴포넌트에 "prop"으로 전달할 수 없습니다. 
+- Astro 컴포넌트는 다시 렌더링되지 않으므로 `.astro` 파일의 변경 사항 저장을 구독할 수 없습니다.
+
+이러한 제한 사항을 이해하고, 여전히 사용 사례를 찾고 있다면 Nano Stores를 사용해 볼 수 있습니다! Nano Stores는 특히 **클라이언트**의 변경 사항에 반응하도록 구축되었다는 점을 기억하세요.
+
+</details>
+
+<details>
+<summary>**🙋 Svelte stores은 Nano Stores과 어떻게 비교되나요?**</summary>
+
+**Nano Stores와 [Svelte stores](https://svelte.dev/tutorial/writable-stores)은 매우 유사합니다!** 실제로 nanostore를 사용하면 Svelte stores에서 사용할 수 있는 구독에 대해 [동일한 `$`를 사용할 수 있습니다](https://github.com/nanostores/nanostores#svelte).
+
+타사 라이브러리를 피하고 싶다면, [Svelte stores](https://svelte.dev/tutorial/writable-stores)는 그 자체로 훌륭한 아일랜드 간 통신 도구입니다. 그럼에도 불구하고, a) ["objects"](https://github.com/nanostores/nanostores#maps) 및 [비동기 상태](https://github.com/nanostores/nanostores#lazy-stores)에 대한 추가 기능이 마음에 들거나 b) Svelte와 Preact 또는 Vue와 같은 다른 UI 프레임워크 간 통신하려는 경우 Nano Store를 선호할 수 있습니다.
+</details>
+
+<details>
+<summary>**🙋 Solid signals는 Nano Stores와 어떻게 비교됩니까?**</summary>
+
+한동안 Solid를 사용해 본 적이 있다면 컴포넌트 외부에서 [signals](https://www.solidjs.com/docs/latest#createsignal) 또는 [stores](https://www.solidjs.com/docs/latest#createstore)를 이동해 보셨을 것입니다. 이는 Solid 아일랜드 간 상태를 공유하는 좋은 방법입니다! 공유 파일에서 signals를 내보내보세요.
+
+```js
+// sharedStore.js
+import { createSignal } from 'solid-js';
+
+export const sharedCount = createSignal(0);
+```
+
+...그리고 `sharedCount`를 가져오는 모든 컴포넌트는 동일한 상태를 공유합니다. 이것이 잘 작동하더라도 a) ["objects"](https://github.com/nanostores/nanostores#maps) 및 [비동기 상태](https://github.com/nanostores/nanostores#lazy-stores)에 대한 추가 기능이 마음에 들거나 b) Solid와 Preact 또는 Vue와 같은 다른 UI 프레임워크 간 통신하려는 경우 Nano Stores를 선호할 수 있습니다.
+</details>
+:::
+
+## Nano Stores 설치
+
+시작하려면 즐겨 사용하는 UI 프레임워크용 도우미 패키지와 함께 Nano Stores를 설치하세요.
+
+<UIFrameworkTabs>
+  <Fragment slot="preact">
+  ```shell
+  npm install nanostores @nanostores/preact
+  ```
+  </Fragment>
+  <Fragment slot="react">
+  ```shell
+  npm install nanostores @nanostores/react
+  ```
+  </Fragment>
+  <Fragment slot="solid">
+  ```shell
+  npm install nanostores @nanostores/solid
+  ```
+  </Fragment>
+  <Fragment slot="svelte">
+  ```shell
+  npm install nanostores
+  ```
+  :::note
+  여기에는 도우미 패키지가 없습니다! Nano Stores는 표준 Svelte stores처럼 사용할 수 있습니다.
+  :::
+  </Fragment>
+  <Fragment slot="vue">
+  ```shell
+  npm install nanostores @nanostores/vue
+  ```
+  </Fragment>
+  <Fragment slot="lit">
+  ```shell
+  npm install nanostores @nanostores/lit
+  ```
+  </Fragment>
+</UIFrameworkTabs>
+
+여기에서 [Nano Stores 사용 안내서](https://github.com/nanostores/nanostores#guide)로 이동하거나 아래 예시를 따라갈 수 있습니다!
+
+## 사용 예 - 전자상거래 장바구니 플라이아웃
+
+세 가지 대화형 요소로 간단한 전자상거래 인터페이스를 구축한다고 가정해 보겠습니다.
+- "add to cart" 제출 양식
+- 추가된 항목을 표시하는 장바구니 플라이아웃
+- 장바구니 플라이아웃 토글
+
+<LoopingVideo sources={[{ src: '/videos/stores-example.mp4', type: 'video/mp4' }]} />
+
+_[**Try the completed example**](https://github.com/withastro/astro/tree/main/examples/with-nanostores) on your machine or online via StackBlitz._
```

Comment:
```suggestion
[**완성된 예시**](https://github.com/withastro/astro/tree/main/examples/with-nanostores)를 컴퓨터에서 또는 StackBlitz를 통해 온라인으로 사용해 보세요!
```
Did you mean to translate this line, too?

Reply:
I missed the translation of this part. Thank you for pointing this out.